src/index.html

```diff
@@ -10,23 +10,39 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
-    <title>Document</title>
+    <title>Search bar for Airbnb</title>
     <link
       rel="stylesheet"
       href="style.css"
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="search-form search-form--big"
+      data-qa="big"
+    >
+      <label class="search-form__label search-form__label--big">
+        <input
+          class="search-form__input search-form__input--big"
+          type="text"
+          data-qa="keypress"
+          placeholder="Try “Los Angeles“"
+        />
+      </label>
+    </form>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="search-form search-form--small"
+      data-qa="small"
+    >
+      <label class="search-form__label search-form__label--small">
+        <input
+          class="search-form__input search-form__input--small"
+          type="text"
+          data-qa="hover"
+          placeholder="Try “Los Angeles“"
+        />
+      </label>
+    </form>
   </body>
```

Comment:
Consider using semantic tags like `section` or `article` to improve SEO and accessibility.